### PR TITLE
Add agg status card pattern

### PR DESCRIPTION
**Pull request by serenamarie125**
Add the pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
Same here with relative path

File: pattern-library/cards/aggregate-status-card/design/overview.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Aggregate Status Card
+
+## Overview
+The aggregate status card shows a total number of objects and an aggregated status for those objects. The most common use case for this pattern is seen in a dashboard view to illustrate the total count and the status of objects in an environment. The mini card alternate can be used in the case when there is less space available for the status card as well as when there is less information to show in the card.
+
+Jump to [Regular Card](#regular-card) or [Mini Card Alternate](#mini-card-alternate)
+
+This is where you should write the overview description of your pattern contribution. This description should include details such as what makes this pattern useful and when it should be used over other, similar patterns. [Links to related patterns](https://www.patternfly.org/pattern-library/other-pattern/other-pattern-design.md) are often found on the overview page.
```
Comment (LHinson) [inline pattern-library/cards/aggregate-status-card/design/overview.md]: This section should be deleted @serenamarie125

**Comment by serenamarie125** [comment]
@LHinson this PR has been updated.  LMK if there is anything else to be addressed!

File: pattern-library/cards/aggregate-status-card/design/design.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Aggregate Status Card
+
+## Header Area
+![Header Callout 1](img/aggregate-status-card-header-callout-1.png)
+![Header Callout 2](img/aggregate-status-card-header-callout-2.png)
+
+  1. **Top Accent:** (optional)**:**
```
Comment (LHinson) [inline pattern-library/cards/aggregate-status-card/design/design.md]: there is an extra ":" ... should be **Top Accent** (optional):

**Comment by serenamarie125** [comment]
@LHinson update PR to remove extra ":"

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**